### PR TITLE
Move gputest expectation helper function to expectationhelper.py

### PR DESCRIPTION
**Pull request by haoxli**

### CODE REVIEW

File: misc/expectationhelper.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    LOCAL_MAINTAINED_EXPECTATIONS = {
+        'third_party/dawn/webgpu-cts/expectations.txt': [
+            'crbug.com/dawn/0000 [ intel win10 ] webgpu:shader,execution,expression,call,builtin,unpack4x8unorm:unpack:inputSource="storage_r" [ Failure ]',
+            'crbug.com/dawn/0000 [ intel win10 ] webgpu:shader,execution,expression,call,builtin,unpack4x8unorm:unpack:inputSource="storage_rw" [ Failure ]',
+            'crbug.com/dawn/0000 [ intel win10 ] webgpu:shader,execution,expression,call,builtin,unpack4x8unorm:unpack:inputSource="uniform" [ Failure ]'
+        ],
+    }
+
+    @staticmethod
+    def update_expectation(expectation_file, append_expectations=None):
```
Comment (gyagp) [inline misc/expectationhelper.py:22]: parameter append_expectations shouldn't be exposed. Whenever we update_expectation, local changes should be appended.

File: misc/expectationhelper.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    EXPECTATION_FILES = {
+        'trace_test': ['content/test/gpu/gpu_tests/test_expectations/trace_test_expectations.txt'],
+        'webgpu_cts_tests': [
+            'third_party/dawn/webgpu-cts/expectations.txt',
+            'third_party/dawn/webgpu-cts/slow_tests.txt'
+        ],
+    }
+
+    # These expected failures are not in expectation files, which will be maintained locally and appended to expectation files.
+    LOCAL_MAINTAINED_EXPECTATIONS = {
```
Comment (gyagp) [inline misc/expectationhelper.py:13]:
```suggestion
    LOCAL_EXPECTATIONS = {
```

File: misc/expectationhelper.py (hunk 1 above)
Comment (haoxli) [reply misc/expectationhelper.py]: Done.

File: misc/expectationhelper.py (hunk 2 above)
Comment (haoxli) [reply misc/expectationhelper.py]: Done.

**Pull request merged**